code/modules/mob/dead/observer/observer.dm

```diff
@@ -387,6 +387,29 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		to_chat(usr, "AntagHud Toggled OFF")
 		M.antagHUD = FALSE
 
+/mob/dead/observer/verb/set_dnr()
+	set name = "Set DNR"
+	set category = "Ghost"
+	set desc = "Prevent your character from being revived."
+
+	if(!isobserver(src)) // Somehow
```

Comment:
Did this actually happen when testing? This should only really occur when somebody fucks up.

Reply:
I'm not sure if it's even possible to be honest, but the other verbs in the file have a check, and it's a verb rather than a proc, so I figured better safe than sorry.